boa/src/syntax/ast/node/field/get_field/mod.rs

```diff
@@ -64,7 +64,7 @@ impl GetField {
 impl Executable for GetField {
     fn run(&self, context: &mut Context) -> Result<Value> {
         let mut obj = self.obj().run(context)?;
-        if obj.get_type() != Type::Object {
+        if obj.typ() != Type::Object {
```

Comment:
```suggestion
        if !obj.is_object() {
```

It's better/cleaner to use the `is_` methods instead of getting the type.

Reply:
Fixed